learn/glossary.md

```diff
@@ -53,18 +93,34 @@ A **Relay Response** is broken down into 2 sections:
 * Response Payload
 * Servicer Signature
 
-Requests are signed by the Client of the Application and responses are signed by the servicing Validator.
+Requests are signed by the client of the application and responses are signed by the servicing validator.
 
-This 1 for 1 signature scheme enables the protocol to validate all parties that participate in the _servicing_ cycle.
+This one-to-one signature scheme enables the protocol to validate all parties that participate in the servicing cycle.
 
 ## Relay Chain
 
-A single database instance, typically a blockchain, provided by a Validator that makes up the Pocket Network decentralized infrastructure.
+Single database instance, typically a blockchain, provided by a Validator that makes up the Pocket Network decentralized infrastructure.
```

Comment:
> But is it really true that Relay Chain ≠ Blockchain?

Technically it can be any consistent, permissionless API that communicates with a decentralized storage layer. At the current point in time, everything we service is a blockchain, but in theory, it could be any decentralized database solution that individuals can spin up clients for.